email_templates/plaintext/shutoff_error.txt.j2

```diff
@@ -0,0 +1,15 @@
+Dear Cloud User {{username}},
+
+We are writing to inform you one or more virtual machines (VMs) associated with your account are currently in a shutoff or errored state and have not been restarted or deleted for the past X days.
```

Comment:
I think we need to add a template variable here for the number of days a VM has been in a given state